app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java

```diff
@@ -24,5 +30,11 @@ public void showResult(float result) {
         visorResult.setText(String.valueOf(result));
     }
 
+    public void showMessageOperation(){
+        Toast.makeText( getContext(), TOAST_ERROR_OPERATOR_MSG, Toast.LENGTH_SHORT).show();
```

Comment:
Put the Toast message into a resource pls

Reply:
done.